.github/actions/minus-one/src/index.mjs

```diff
@@ -0,0 +1,44 @@
+// @ts-check
+import { context } from '@actions/github'
+import { info, setFailed } from '@actions/core'
+import { graphql } from '@octokit/graphql'
+
+const offTopicComments = ['+1', '👍', 'same issue', 'any updates']
+
+async function run() {
+  try {
+    if (!process.env.GITHUB_TOKEN) return
+
+    const { comment } = context.payload
+    if (!comment) return
+
+    const { node_id: subjectId, body } = comment
+
+    const bodyLower = body.toLowerCase()
+    if (offTopicComments.some((comment) => bodyLower === comment)) {
```

Comment:
This check is very rudimentary. We have to make sure we only mark truly unhelpful comments. Open to suggestions on how we can catch more comments like these reliably.

Reply:
https://github.com/vercel/next.js/pull/52866#discussion_r1267968052